storage/client/queries/queries.go

```diff
@@ -455,7 +476,7 @@ const (
 			tokens.symbol,
 			tokens.decimals,
 			tokens.total_supply,
-			CASE -- NOTE: There are two queries that use this CASE via copy-paste; edit both if changing.
+			CASE -- NOTE: There are three queries that use this CASE via copy-paste; edit both if changing.
```

Comment:
:grinning: :see_no_evil:  this is ripe for a tiny refactor; by the book, we should be doing this conversion (which happens for the sake of openapi) in Go anyway. Not in this PR though :)